rrfs-test/validated_yamls/templates/obtype_config/msonet_uv_288.yaml

```diff
@@ -1,210 +0,0 @@
-     - obs space:
```

Comment:
It looks like you committed the removal first and then add the renamed file? We expect to see the differences between the old and new file here but it is not.

Reply:
@guoqing-noaa, sorry about that if I did something out of order. I thought I committed my changes to the file with the "uv" name and then committed the name change. I was thinking when you change a file name it doesn't like to show the differences--seems to be the case when I do it. Maybe there is a better move order that I should learn.